core/store/models/common.go

```diff
@@ -242,6 +242,11 @@ func (w *WebURL) UnmarshalJSON(j []byte) error {
 	if err != nil {
 		return err
 	}
+	// handle no url case
+	if len(v) == 0 {
```

Comment:
Is this causing a test to fail? I feel like it shouldn't be needed, but I could be wrong.

Reply:
We need that to avoid throwing an error when there is no url. If the bytes array is empty then we'd throw an error when we `url.ParseRequestURI(v)`.